google/cloud/spanner/value_test.cc

```diff
@@ -163,6 +165,17 @@ TEST(Value, SpannerArray) {
   using ArrayInt64 = std::vector<std::int64_t>;
   using ArrayDouble = std::vector<double>;
 
+  ArrayInt64 const empty = {};
+  Value const ve(empty);
+  EXPECT_EQ(ve, ve);
+  EXPECT_TRUE(ve.is<ArrayInt64>());
+  EXPECT_FALSE(ve.is_null<ArrayInt64>());
+  EXPECT_FALSE(ve.is<ArrayDouble>());
+  EXPECT_EQ(empty, static_cast<ArrayInt64>(ve));
```

Comment:
Perhaps the conversion operator test should be the last of the bunch (i.e., after the get<T>.ok() and *get<T>())?

Reply:
SGTM. Done. Looks nicer this way, and is a bit more logical.